cookbook/entities/creating_entities.rst

```diff
@@ -0,0 +1,191 @@
+
+    /**
+     * @ORM\Entity()
+     * @ORM\Table(name="app_task")
+     */
+    class Task
+    {
+        /**
+         * @ORM\Id()
+         * @ORM\GeneratedValue(strategy="AUTO")
+         * @ORM\Column(type="integer")
+         *
+         * @var int
+         */
+        private $id;
+
+        /**
+         * @ORM\Column(type="string")
+         *
+         * @var string
+         */
+        private $subject;
+
+        /**
+         * @ORM\Column(type="text")
+         *
+         * @var string
+         */
+        private $description;
+
+        /**
+         * @ORM\Column(type="datetime", name="due_date")
+         *
+         * @var \DateTime
+         */
+        private $dueDate;
+
+        /**
+         * @ORM\ManyToOne(targetEntity="Priority")
+         * @ORM\JoinColumn(name="task_priority_id", onDelete="SET NULL")
+         *
+         * @var Priority
+         */
+        private $priority;
+
+        public function getId()
+        {
+            return $this->id;
+        }
+
+        public function getSubject()
+        {
+            return $this->subject;
+        }
+
+        public function setSubject($subject)
+        {
+            $this->subject = $subject;
+        }
+
+        public function getDescription()
+        {
+            return $this->description;
+        }
+
+        public function setDescription($description)
+        {
+            $this->description = $description;
+        }
+
+        public function getDueDate()
+        {
+            return $this->dueDate;
+        }
+
+        public function setDueDate(\DateTime $dueDate)
+        {
+            $this->dueDate = $dueDate;
+        }
+
+        public function getPriority()
+        {
+            return $this->priority;
+        }
+
+        public function setPriority(Priority $priority)
+        {
+            $this->priority = $priority;
+        }
+    }
+
+Priorities for tasks will be configurable. Thus, they are modeled as separate entities:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Entity/Priority.php
+    namespace AppBundle\Entity;
+
+    use Doctrine\ORM\Mapping as ORM;
+
+    /**
+     * @ORM\Entity()
+     * @ORM\Table(name="app_task_priority")
+     */
+    class Priority
+    {
+        /**
+         * @ORM\Id()
+         * @ORM\GeneratedValue(strategy="AUTO")
+         * @ORM\Column(type="integer")
+         *
+         * @var int
+         */
+        private $id;
+
+        /**
+         * @ORM\Column(type="string", unique=true)
+         *
+         * @var string
+         */
+        private $label;
+
+        public function getId()
+        {
+            return $this->id;
+        }
+
+        public function getLabel()
+        {
+            return $this->label;
+        }
+
+        public function setLabel($label)
+        {
+            $this->label = $label;
+        }
+    }
+
+After you have modeled your entities, you need to update the database schema. This can be done by
```

Comment:
Please add note that doctrine:schema:update command should be used only during development. Developer should write migrations (documentation you can find here https://github.com/orocrm/platform/tree/master/src/Oro/Bundle/MigrationBundle#database-structure-migrations) to update db schema on live server.